frontend/app/src/components/Sidebar/styled-components.ts

```diff
@@ -52,6 +52,7 @@ export const StyledSidebar = styled.section<StyledSidebarProps>(
       // Nudge the sidebar by 2px so the header decoration doesn't go below it
       top: adjustTop ? theme.sizes.headerDecorationHeight : theme.spacing.none,
       backgroundColor: theme.colors.bgColor,
+      color: theme.colors.bodyText,
```

Comment:
Can you also add: `colorScheme` here and maybe a comment for why we set the color here, e.g.: see the deprecated sidebar PR here:

https://github.com/streamlit/streamlit/pull/10368/files#diff-d091b0d99a8ed4c4f4308949ba8b0d5eccc982513aa72ce1bd300e04696ae2d0R57